GPyOpt/objective_examples/experimentsNd.py

```diff
@@ -28,7 +28,7 @@ def __init__(self,input_dim, bounds=None, sd=None):
     def f(self,X):
         X = reshape(X,self.input_dim)
         n = X.shape[0]
-        fval = (X*np.sin(X) + 0.1*X).sum(axis=1) 
```

Comment:
This is a good catch but it breaks some of the tests. Would you mind to do as follows:
- create a new objective, that you can call alpine_modified
- call this new function from the test that are broken with the changes.
This will correct the general bug and will keep the test as they are.

Reply:
Alternatively, and actually preferably, you could just fix the tests with this new, correct function implementation. Let us know if you need help doing that.

Reply:
Agree @apaleyes! this is probably better!

Reply:
I am not familiar with the automatic testing flow, how can I firstly establish the testing flow on my personal PC?

Reply:
@Alaya-in-Matrix GPyOpt uses nose to run its tests. Just install nose with pip, go to the GPyOpt folder and run `nosetests`, that will discover and run all unit tests in GPyOpt.